### PR TITLE
add pydap.__version__ capability (#133)

### PR DESCRIPTION
**Pull request by tomkralidis**
As per #133

### CODE REVIEW

File: src/pydap/__init__.py [hunk 1]
```diff
@@ -3,3 +3,5 @@
 '''
 
 __import__('pkg_resources').declare_namespace(__name__)
+
+__version__ = __import__('pkg_resources').get_distribution(__name__).version
```
Comment (ocefpaf) [inline src/pydap/__init__.py]: I'm not a fan of `pkg_resources`, it is slow and hackish.
Could we try something like versioneer? Or maybe define the version in `__init__.py` and parse it in `setup.py`?

**Comment by tomkralidis** [comment]
@ocefpaf fixed.  Travis errors (Python 3.5) are unrelated. This PR requires squashing when merging.

**Comment by ocefpaf** [comment]
LGTM. @jameshiebert this one is an easy merge :wink:

**Comment by jhamman** [comment]
this looks good. If you are looking for a standard (likely more robust) approach, you may want to checkout [versioneer](https://github.com/warner/python-versioneer).

**Comment by tomkralidis** [comment]
Thanks @jhamman. versioneer looks great, but I'd prefer to not include another dependency.

**Comment by jhamman** [comment]
Okay, for clarity sake though, versioneer is not an external dependency.

**Comment by ocefpaf** [comment]
:+1: for `versioneer`, it is a one time thing and you never have to worry about versions again :stuck_out_tongue_winking_eye:

**Pull request merged**